app/models/plugins/fundraiser.rb

```diff
@@ -7,7 +7,10 @@ class Plugins::Fundraiser < ActiveRecord::Base
   DEFAULTS = { title: 'Donate now' }
 
   def liquid_data(supplemental_data={})
-    bands = donation_band.present? ? donation_band.internationalize.to_json : "null"
+    donation_band_name = supplemental_data.has_key?(:url_params) ? supplemental_data[:url_params][:donation_band] : ''
```

Comment:
my final fyi, it's fine to pass `nil` to `find_by`, so a ternary operator wouldn't be necessary.